### ISSUE TITLE
Wrong work within containers with position: fixed 

### ISSUE THREAD

**Issue by bondarenkokv** [Amphiluke/floating-scroll#8]
Hey guys, this is really awasome plugin, thank you so mush for it!
But unfortunatelly I got issue when scroll added to a block within scrollable container with position: fixed. Looking through the code if found next method:
```
var wnd = window,
getMaxVisibleY = function () {return wnd.pageYOffset + wnd.innerHeight;};
```
It's used in _checkVisibility()_ . As you see this max visible Y is calculated according to page's offset but actually it was overriden by fixed block's position (including its own scroll).
The single solution I can see it's for parrent container with position fixed instead of window here + listening to scrolling of it. 

Html will be looking like this: https://jsfiddle.net/12x4t50h/

**Comment by Amphiluke** [comment]
Hey @bondarenkokv,
Thank you for the report! This is rather uncommon use case you know, and the plugin wasn't intended to make a provision for such a case. I will consider this enhancement for the next release.

**Comment by bondarenkokv** [comment]
Thanks for replay @Amphiluke. 
I understood you. Actually it's not such rare case because, for example, popus mostly are blocks with position fixed. But anyway your plugin still being cool, respect you!:)

**Comment by Amphiluke** [comment]
The feature has been implemented in [v2.3.0](https://github.com/Amphiluke/floating-scroll/releases/tag/v2.3.0).
Please find out the usage details in the section [Special cases](https://github.com/Amphiluke/floating-scroll#special-cases) of the docs. A [live demo](https://amphiluke.github.io/floating-scroll/#demo-popup) is also available on the plugin web page.
